R/rules-line-breaks.R

```diff
@@ -447,3 +447,14 @@ remove_empty_lines_after_opening_and_before_closing_braces <- function(pd) {
 
   pd
 }
+
+
+reduce_extra_blank_lines_between_scopes <- function(pd, allowed_blank_lines = 2L) {
```

Comment:
I believe what you call _scope_ is called top level expressions in this repo (just full text search for it). Can you re-name this function and PR to that effect?